Scripts/extras/pull_and_build_from_git.py

```diff
@@ -919,6 +932,9 @@ def execute(self):
             # Build the package
             self.build_for_platform()
 
+            # Copy extra files specified in the build config
+            self.copy_extra_files()
+
```

Comment:
Also, this functionality should go inside the `self.generate_package_info()` function instead of here.

Reply:
We need the files to be copied and available before self.test_package()